tools/dune_rule_gen/coq_rules.mli

```diff
@@ -0,0 +1,40 @@
+(************************************************************************)
+(* This file is licensed under The MIT License                          *)
+(* See LICENSE for more information                                     *)
+(* (c) MINES ParisTech 2018-2019                                        *)
+(* (c) INRIA 2020-2022                                                  *)
+(* Written by: Emilio Jesús Gallego Arias                               *)
+(* Written by: Rudi Grinberg                                            *)
+(************************************************************************)
+
+(** theory kind *)
+module Boot_type : sig
+
+  type t =
+      Stdlib
+    (** Standard library *)
+    | NoInit
+    (** Standalone library (without Coq's stdlib) *)
+    | Regular of Path.t option
+    (** Regular library, path controls where the Coq stdlib is *)
+
+end
+
+(** [gen_vo_rules root_lvl native tname boot] Builds dune rules for
+   the *current* directory, assuming that we will do [-R . tname]. The
+   parameter [boot] controls the kind of the current
+   theory. [root_lvl] tells the rule generator how many levels up the
+   root Coq sources are. *)
+val gen_vo_rules :
+  root_lvl:int
+  -> native:bool
+  -> tname:string list
+  -> boot:Boot_type.t
+  -> Dune_file.Rule.t Dir_info.t
```

Comment:
Yes, the native flag is here because the coqnative support is not in dune master yet (there's a PR), I agree we should support both methods maybe? This one is faster.

Reply:
Both methods will be supported in the PR.